Sources/Rendering/Core/ImageResliceMapper/example/index.js

```diff
@@ -189,6 +209,19 @@ for (let idx = 0; idx < slabTypes.length; ++idx) {
     renderWindow.render();
   };
 }
+const sliceFunction = document.querySelectorAll('.spd');
+for (let idx = 0; idx < sliceFunction.length; ++idx) {
+  const st = sliceFunction[idx];
+  st.onchange = (e) => {
+    if (e.target.value === 'plane') {
+      mapper.setSlicePolyData(null);
```

Comment:
Looks like the mapper switches back to Plane slicing if the polydata is set to `null`. This kinda feels too subtle. Class design wise, would it be better to have an explicit flag or integer to set the modes to implicit function vs polydata slicing? or handle the type internally and only have a single setter like `setSlicingGeometry`?

Reply:
This PR doesn't change that behavior and that's documented here: https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/Core/ImageResliceMapper/index.d.ts#L206-L214

When designing the mapper, I considered adding a flag to switch between the two but figured preferring the polydata input should be fine for most cases.